.vscode/settings.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
shouldn't the entire .vscode folder be in .gitignore?

Reply:
We need our vscode settings to be consistent for all members